src/streamlink/plugins/welt.py

```diff
@@ -1,4 +1,5 @@
 """
+$description Two 24-hour channels covering news & features and video content from Die Welt, based in Germany.
```

Comment:
> N24 Doku ... which is a one-hour delayed timeshift channel of Welt
https://en.wikipedia.org/wiki/Welt_(German_TV_channel)

wound't count it as two channels

> content from Die Welt

might be the same name, but the description is weird.

It is just a news channel with documentaries inbetween